tools/monitoring/README.md

```diff
@@ -28,14 +28,21 @@ monitoring_service file to make any updates.
 
 ### Update Image
 
-1. `images/monitoring/Makefile` Commands to build and deploy the monitoring
-   images.
+1. Build and deploy the monitoring images.
+   ```bash
+   cd images/monitoring/
+   make build
+   make push_versioned
+   ```
+
+1. Update `tools/monitoring/gke_deployment/monitoring_service.yaml` to use the image tag created in previous step
 
 1. Update to use the latest image on GKE
 
    ```bash
    kubectl rollout restart deployment.apps/monitoring-app
```

Comment:
The rollout restart would work only if the tag remains the same. (also i think the command may be deprecated)
To use the latest image, create a new PR updating [monitoring.yaml](https://github.com/knative/test-infra/blob/master/tools/monitoring/gke_deployment/monitoring_service.yaml#L43), and run `kubectl apply -f <monitoring_service.yaml>` on the monitoring cluster after that's merged.

Reply:
Merging is not required to test out the result with `kubectl apply -f <monitoring_service.yaml>`, right?

Reply:
do you mean you'd like to run `kubectl apply -f <monitoring_service.yaml>` without the change merged into the service? 

If that's the case, I would say merging is still required. Otherwise, you'll be saying merging changes in made in the monitoring_service.yaml can be changed without going through proper code reviews. 

If you're interested in fully testing changes, spin up a cluster and deploy the service on your sandbox project.

Reply:
I see. That's a practice which holds for other projects as well. I guess we don't need to include it here.